internal/refreshtoken/refresh_token.go

```diff
@@ -76,17 +81,26 @@ func FromResource(res boundary.Resource, grantsHash []byte) *Token {
 	}
 }
 
-// Refresh refreshes a token's updated time
+// Refresh refreshes a token's updated time. It accounts for database
+// transaction inaccuracies by subtracting UpdatedTimeBuffer from the
```

Comment:
I don't think "inaccuracies" is the right word here. It is the nature of a [MVCC](https://www.postgresql.org/docs/current/mvcc-intro.html). There may have been items that have an older update time than `updatedTime` that where not visible at the time `updatedTime` was returned.

Reply:
Reworded this slightly, please take a look.